modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/configuration/DLPreviewLimitConfigurationProvider.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.document.library.configuration;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * @author Marco Galluzzi
+ */
+@ProviderType
+public interface DLPreviewLimitConfigurationProvider {
+
+	public static final long UNLIMITED_MAX_FILE_SIZE = -1;
+
+	public static final int UNLIMITED_MAX_NUMBER_OF_PAGES = 0;
```

Comment:
I think it is better to add these to a `*Constants` class. This is not wrong, but it is a bit unconventional (for LR standards); better chances of this going through Brian if we follow the usual path.

Reply:
I will move both values to Constants. About the question "why two different values to mean unlimited?". The new settings page is the result of joining two different settings, each one having different values for unlimited.

<img width="1252" alt="Captura de pantalla 2023-09-17 a las 16 11 29" src="https://github.com/liferay-lima/liferay-portal/assets/19521293/7c6593b8-a23d-4abc-baba-02d443233ddd">

I've created a [Slack question](https://liferay.slack.com/archives/GQBCVUPBQ/p1694963298661749) for UX/Design team to improve that